packages/calcite-components/src/components/action-menu/action-menu.tsx

```diff
@@ -323,9 +323,11 @@ export class ActionMenu implements LoadableComponent {
   render(): VNode {
     return (
       <Fragment>
```

Comment:
Similar to https://github.com/Esri/calcite-design-system/pull/9065/files#r1558143540, `<Fragment>` is no longer needed.